pyproject.toml

```diff
@@ -55,3 +41,5 @@ line-length = 79
 [tool.ruff.per-file-ignores]
 # Lots of pytest implicitly injected attributes in conftest-patch.py
 "python/cudf/cudf/pandas/scripts/conftest-patch.py" = ["F821"]
+"python/cudf/cudf/pandas/scripts/*" = ["D201", "D204", "D206", "D207", "D208", "D209", "D210", "D211", "D214", "D215", "D300", "D301", "D403", "D405", "D406", "D407", "D408", "D409", "D410", "D411", "D412", "D414", "D418"]
```

Comment:
Do you want to just ignore D*?

Reply:
Nice, yeah replacing with just `["D"]` is cleaner